script/tasks/diamondSyncDEXs.sh

```
@@ -110,23 +154,46 @@ function diamondSyncDEXs {
+            echo "[$NETWORK] Error: Token contracts detected in new addresses"
+            echo "[$NETWORK] Token addresses: ${TOKEN_CONTRACTS[*]}"
+          } >> "$FAILED_LOG_FILE"
+          return
+        fi
+      fi
+    fi
+
     # Add missing DEXs
     if [[ ! ${#NEW_DEXS[@]} -eq 0 ]]; then
       ADDRESS_STRING=$(printf "%s," "${NEW_DEXS[@]}")
       PARAMS="[${ADDRESS_STRING%,}]"
```

Shellcheck (0.10.0)

[warning] PARAMS appears unused. Verify use (or export if used externally).

(SC2034)